Cake.Recipe/Content/gitversion.cake

```diff
@@ -16,6 +16,29 @@ public class BuildVersion
             throw new ArgumentNullException("context");
         }
 
+        var cakeVersion = typeof(ICakeContext).Assembly.GetName().Version.ToString();
+
+        try
+        {
+            var rootPath = BuildParameters.RootDirectoryPath;
+            rootPath = context.GitFindRootFromPath(rootPath);
+        }
+        catch (LibGit2Sharp.RepositoryNotFoundException rnfe)
```

Comment:
Instead of having a try/catch block, we could check if there is a `.git` directory in the root.

Reply:
Isn’t that the purpose of this method? Why would we additionally check for the .git folder?

Reply:
yeah, you are right.
I read the context wrong here 😕